scripts/tests/test_validate_docstrings.py

```diff
@@ -1052,6 +1052,13 @@ def test_raises_for_invalid_attribute_name(self, invalid_name):
         with pytest.raises(AttributeError, match=msg):
             validate_docstrings.Docstring(invalid_name)
 
+    @pytest.mark.parametrize('name', [
+        'pandas.Series.str.isdecimal', 'pandas.Series.str.islower'])
+    def test_encode_content_write_to_file(self, name):
+        # GH25466
+        docstr = validate_docstrings.Docstring(name).validate_pep8()
+        assert len(list(docstr)) == 0
```

Comment:
in Python to check if a list is empty, `not list(docstr)` is preferred over `len(list(docstr)) == 0`

Reply:
since it's not obvious what `validate_pep8()` returns (feels like it could be a bool...). I'd add a short comment before the assert explaining. e.g.: `# the list of pep8 errors should be empty`